frontend/src/pages/rooms/[id]/index.tsx

```diff
@@ -32,6 +34,10 @@ export default function RoomPage() {
   return (
     <Wrapper>
       <MessageDisplay messages={messages} />
+      <Tutorial
+        isVisible={!!room && !isTutorialDone}
+        moodPercentage={moodPercentage}
+      />
```

Comment:
IMOだけど、コンポーネントの表示/非表示を切り替えるのはコンポーネントの呼び出し元で管理しても良いのかなと思いました！

Roomが管理してる`moodPercentage`の状態で、チュートリアルの終了状態`isTutorialDone`が変わる
→ その状態を[id]ページで購読している
→ 終了したら状態の変更が[id]ページの方に伝わって、そこから何らかのアクションを行う（フェードアウトさせたり）
みたいなイメージ！（？）
```suggestion
イメージ
    <Fadeout>
      {isVisible && (
        <Tutorial
          moodPercentage={moodPercentage}
        />
      )}
    </Fadeout>
```

Reply:
@sts11142 たしかに、その実装良さそう！
リファクタすることがあったらやってみる！